### ISSUE TITLE
Moon Tester Tool blocked by PM 29.4.2.1

### ISSUE THREAD

**Issue by PotentialTumbleweed** [JustOff/moon-tester-tool#14]
Created some new profiles with Pale Moon on Windows 10. Discovered that I could not install Moon Tester Tool. In my older profiles, I had MTT installed already, and they have not been disturbed.

**Comment by JustOff** [comment]
Duplicate of #10.
